includes/admin/class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -336,43 +336,46 @@ public function data_table_header() {
 			$selected_course = (int) $_GET['filter_by_course_id']; // phpcs:ignore WordPress.Security.NonceVerification
 		}
 		?>
-		<div class="tablenav top">
-			<div class="alignleft actions bulkactions">
-				<div id="sensei-bulk-learner-actions-modal" style="display:none;">
-					<?php $this->render_bulk_actions_form( $courses ); ?>
-				</div>
-				<?php
-				echo wp_kses(
-					$this->render_bulk_action_select_box(),
-					array(
-						'option' => array(
-							'value' => array(),
-						),
-						'select' => array(
-							'id'   => array(),
-							'name' => array(),
-						),
-					)
-				);
-				?>
-				<button type="submit" id="sensei-bulk-learner-actions-modal-toggle" class="button button-primary action"><?php echo esc_html__( 'Select Courses', 'sensei-lms' ); ?></button>
+		<div class="sensei-bulkactions-wrapper">
+			<div class="alignleft actions bulkactions sensei-bulkactions-container">
+			<div id="sensei-bulk-learner-actions-modal" style="display:none;">
+				<?php $this->render_bulk_actions_form( $courses ); ?>
 			</div>
-			<div class="alignleft actions">
-				<form action="" method="get">
+			<div class="sensei-wp-table-filters">
+				<div>
```

Comment:
Do we need this extra `div` when we have the one above?

Reply:
We do, to wrap select and the button, they are one item when applying flex